### PR TITLE
RX python script cleanups, proper file naming for future TX capabilities

### PR DESCRIPTION
**Pull request by krippendorf**
Thought the script should be renamed before anyone integrating it into own scripts, because it'll provide TX functions as well. That's why this pull request is a bit small.

### CODE REVIEW

**Comment by pavel-demin** [comment]
Thanks for this update.

I have a couple of comments/suggestions:

1. I've just done some reading about comments in the Python code and discovered that it's normally not recommended to use string literals as comments. The main arguments are that it's not standard and that there could be some side effects. So, could you please replace the string literals with # prefixed comments?

2. I think that the python sub-directory is one sub-directory too much :-) I'd prefer to keep all the clients in one directory called clients. If, one day, there are too many clients in this directory, then we'll move it to a separate GitHub repository.

3. I think that it would be better to keep TX and RX clients in separate files.

**Comment by krippendorf** [comment]
agreed. Will update. 

>  If, one day, there are too many clients in this directory, then we'll move it to a separate GitHub repository.

Sounds good.

File: projects/sdr_transceiver/clients/remote_rx.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+        self.data_socket.close()
+        self.control_socket.close()
+        sys.stdout.flush()
+
+        sys.stderr.write("bye\n")
+        sys.exit(0)
+
+    def start_rx(self, freq, rate_index, corr):
+        """
```
Comment (krippendorf) [inline projects/sdr_transceiver/clients/remote_rx.py]: According to my linter that style should be fine

File: projects/sdr_transceiver/clients/remote_rx.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        Connects to the Red Pitaya Transeiver server and writes the data to stdout
+        :param freq: Frequency in Hz
+        :param rate_index: Sample rate index
+        :param corr: correction in ppm
+        :return:
+        """
+        # Setup control socket
```
Comment (krippendorf) [inline projects/sdr_transceiver/clients/remote_rx.py]: changed others to # style comments

**Comment by pavel-demin** [comment]
Great! Thanks for the updates!

**Pull request merged**